vignettes/adrs.Rmd

```diff
@@ -314,6 +312,57 @@ adrs <- adrs %>%
   )
 ```
 
+```{r, echo=FALSE}
+dataset_vignette(
+  adrs,
+  display_vars = exprs(USUBJID, AVISIT, PARAMCD, AVALC, ADT, ANL01FL, ANL02FL)
+)
+```
+
+### Select Source Assessments for Parameter derivations
+
+For most parameter derivations the post-baseline overall response assessments up
+to and including first PD are considered.
+```{r}
+ovr <- filter(adrs, PARAMCD == "OVR" & ANL01FL == "Y" & ANL02FL == "Y")
+```
+
+```{r, echo=FALSE}
+dataset_vignette(
```

Comment:
the 3 patients shown here aren't great examples in the rendered vignette as all their first assessments were `PD` so they just get those records kept, maybe choose a better example patient to include to show this?

Reply:
Should be solved by the new test data (https://github.com/pharmaverse/admiralonco/issues/226).

Reply:
ok @bundfussr  will trust you to double check this later

Reply:
Meanwhile, the new data is used, i.e., it should look better now.